install-scripts/linux/skm-install-linux.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+set -eu
+GIT_SKM_REPO=https://github.com/splashkit/skm.git
+MARK_CHECK="\xE2\x9C\x94"
+MARK_CROSS="\xE2\x9C\x95"
+
+HOME_PATH=~
+INSTALL_PATH="${HOME_PATH}/.splashkit"
+
+function has_git() {
+    git --help 2>&1 > /dev/null
+    return $?
+}
+
+if ! has_git; then
+    echo -e "\t$MARK_CROSS Pre-requisite 'git' is not installed."
+    if which apt > /dev/null 2>&1; then
+        sudo apt-get install git
+    elif which pacman > /dev/null 2>&1; then
+        sudo pacman -S git
+    elif which dnf > /dev/null 2>&1; then
+        sudo dnf install git
+    else
+        echo -e "\t$MARK_CROSS Unable to determine your package manager. Please install git and try again."
+    	exit 1
+    fi
+
+	if ! has_git; then
+        echo -e "\t$MARK_CROSS Failed to install pre-resuite 'git'. Please install git and try again."
+    	exit 1
+	fi
+fi
+
+echo "Checking for an existing version of SplashKit..."
+if [ -d "${INSTALL_PATH}" ]; then
+    echo -e "\t$MARK_CROSS SplashKit is already installed!"
+    read -p "Would you like to completely re-install SplashKit? (Y/N): " -n 1 -r < /dev/tty
+    echo ""
+    if [[ $REPLY =~ [Yy]$ ]]; then
+        D=`date +%y%m%d-%H%M%S`
```

Comment:
**suggestion (non-blocking):** I can see that we use the same construct to get the current date in the `skm-install-mac.sh` script. However, I'd suggest using subshell `$()` instead of backticks (see https://github.com/koalaman/shellcheck/wiki/SC2006). We can do a clean up for both in a follow-up PR.